sdk/tests/deqp/functional/gles3/es3fFboTestCase.js

```diff
@@ -135,7 +135,7 @@ var DE_ASSERT = function(x) {
             /** @const @type {goog.NumberArray} */ var supportedSampleCounts = es3fFboTestCase.querySampleCounts(sizedFormat);
             var supported = Array.prototype.slice.call(supportedSampleCounts);
             if (supported.indexOf(numSamples) == -1) {
-                if (numSamples >= 2) {
+                if (minSampleCount == 0 || numSamples > gl.getParameter(gl.MAX_SAMPLES)) {
```

Comment:
I think the native deqp is wrong. If these floating point formats can not be multisampled, the native deqp will fail. 
This is not correct. You know, these floating point formats are not required to be multisampled. It is only optional.